ledger-service/jwt/src/main/scala/com/digitalasset/jwt/JwksVerifier.scala

```diff
@@ -64,17 +64,17 @@ class JwksVerifier(
     * On a cache miss, creates a new verifier by fetching the public key from the JWKS URL. */
   private[this] def getCachedVerifier(keyId: String): Error \/ JwtVerifier = {
     if (keyId == null)
-      -\/(Error('getCachedVerifier, "No Key ID found"))
+      -\/(Error(Symbol("getCachedVerifier"), "No Key ID found"))
```

Comment:
Did they drop the symbol literals?

Reply:
They still work in 2.13 but emit a warning as being deprecated